### DISCUSSION TITLE
打开超时和 针对电信的m3u 有吗？

### DISCUSSION THREAD

**Discussion by sendme365** [HerbertHe/iptv-sources#25] [General]
刚开始玩IPTV ，选择了几个ipv6或者v6后缀的文件的url 地址放入到播放软件后，不知道为什么都是超时。然后家里是电信的宽带还有软路由。另外哪个channel列表里的m3u是针对电信的 m3u?

**Comment by HerbertHe** [comment]
emmm影响因素有很多，需要一个个因素排除。另，不支持针对任何运营商
